errors.go

```diff
@@ -61,10 +58,10 @@ func (ve ValidationErrors) Translate(ut ut.Translator) ValidationErrorsTranslati
 
 	trans := make(ValidationErrorsTranslations)
 
-	var fe *fieldError
+	var fe FieldError
 
 	for i := 0; i < len(ve); i++ {
-		fe = ve[i].(*fieldError)
+		fe = ve[i].(FieldError)
```

Comment:
It is not necessary to make this cast here. The `ve` is already a slice of `FieldError`.
```suggestion
		fe = ve[i]
```

Reply:
Thanks for your review. I have removed it with test.